report/src/main/webapp/app/services/endpointConfiguration.service.js

```diff
@@ -18,15 +18,21 @@
 define(['angularAMD'], function (angularAMD) {
   'use strict';
   angularAMD.factory('endpointConfiguration', function () {
+    var domain = 'http://aet-vagrant';
```

Comment:
This can't be hardcoded! :)
Otherwise it won't work on different environments than `http://aet-vagrant`.
This is why `getUrl` returns relative url `/api`.
Please don't change this. Your domain is exactly the domain, at which this report is displayed.
Look how it is done in [`metadataEndpoint`](https://github.com/Cognifide/aet/blob/master/report/src/main/webapp/app/services/metadataEndpoint.service.js#L61) to call the endpoint without domain provided.